# HG changeset patch
# User Rik <rik@octave.org>
# Date 1375332846 25200
#      Wed Jul 31 21:54:06 2013 -0700
# Node ID 1f2c232feaffaa75301e03e9df757f7519cca24b
# Parent  fd9a8b1a8768efbba4ca6a9a5fff1d826029656b
figure.m: Don't initialize figure to 1 as it may already exist as hidden figure.

* scripts/plot/gcf.m: Call figure() without arguments to generate a
new auto-numbered figure.

diff --git a/scripts/plot/gcf.m b/scripts/plot/gcf.m
--- a/scripts/plot/gcf.m
+++ b/scripts/plot/gcf.m
@@ -29,30 +29,30 @@
 ## fig = gcf ();
 ## set (fig, "visible", "off");
 ## @end group
 ## @end example
 ##
 ## @noindent
 ## plots a sine wave, finds the handle of the current figure, and then
 ## makes that figure invisible.  Setting the visible property of the
-## figure to @code{"on"} will cause it to be displayed again.
+## figure to "on" will cause it to be displayed again.
 ## @seealso{gca, gco, gcbf, gcbo, get, set}
 ## @end deftypefn
 
 ## Author: jwe, Bill Denney
 
 function h = gcf ()
 
   if (nargin == 0)
     h = get (0, "currentfigure");
     if (isempty (h) || h == 0)
       ## We only have a root figure object, so create a new figure
       ## object and make it the current figure.
-      h = figure (1);
+      h = figure ();
     endif
   else
     print_usage ();
   endif
 
 endfunction
 
 
